FirstSolution/Part17Task1P2/Rhombus.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace Part17Task1P2
+{
+    public class Rhombus : ISimpleNAngle
+    {
+        private const int numberOfSides = 4;
+
+        public Rhombus(double floor, double angleBetweenFloorAndAdjacentSide)
+        {
+            if (
+                angleBetweenFloorAndAdjacentSide > 0 &&
+                angleBetweenFloorAndAdjacentSide < 180 &&
+                floor > 0
+                )
```

Comment:
Fix if statement arguments formatting. Don't leave space between brackets and arguments("if (angle...")

Reply:
Done